web/models.py

```
@@ -2578,6 +2578,97 @@ def __str__(self):
+
+
+class PDFSubmission(models.Model):
+    """Model for storing PDF documents submitted by students for review."""
+
+    STATUS_CHOICES = [
+        ("submitted", "Submitted"),
+        ("under_review", "Under Review"),
+        ("reviewed", "Reviewed"),
+        ("needs_revision", "Needs Revision"),
+        ("approved", "Approved"),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)